pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/SubscribeRateLimiter.java

```diff
@@ -51,7 +51,7 @@ public SubscribeRateLimiter(PersistentTopic topic) {
         subscribeRateLimiter = new ConcurrentHashMap<>();
         this.executorService = brokerService.pulsar().getExecutor();
         // get subscribeRate from topic level policies
-        this.subscribeRate = Optional.ofNullable(brokerService.getTopicPolicies(TopicName.get(this.topicName)))
+        this.subscribeRate = Optional.ofNullable(topic.getTopicPolicies())
```

Comment:
Rather than returning null and having downstream user having to check for that, it would be better to return `Optional<Policies>` directly.

Reply:
I agree that `Optional` is be better. I'll refactor this PR to use `Optional`.